target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-ad7200.dts

```diff
@@ -0,0 +1,422 @@
+#include "qcom-ipq8064-v1.0.dtsi"
+#include <dt-bindings/input/input.h>
```

Comment:
please add ```#include <dt-bindings/gpio/gpio.h>``` here as well and use GPIO_ACTIVE_LOW and GPIO_ACTIVE_HIGH. Have a look at https://git.lede-project.org/?p=lede/mkresin/staging.git;a=commitdiff;h=ea5b162e1124d6861555c034d8946d487881c425 for an example.